src/modules/shlvl.rs

```diff
@@ -0,0 +1,40 @@
+use std::env;
+
+use super::{Context, Module, SegmentConfig};
+
+use crate::config::RootModuleConfig;
+use crate::configs::shlvl::ShLvlConfig;
+
+const SHLVL_ENV_VAR: &str = "SHLVL";
+
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let shlvl = get_shlvl_value()?;
+
+    let mut module = context.new_module("shlvl");
+    let config: ShLvlConfig = ShLvlConfig::try_load(module.config);
+
+    if config.disabled {
```

Comment:
```suggestion
    if config.disabled || shlvl < config.threshold {
```

Use early return to avoid indenting the main portion of the function.